### ISSUE TITLE
Barra de progresso de etapas do Projeto

### ISSUE THREAD

**Issue by nitaibezerra** [mateuswetah/Projeto-Salic#20]
Todo projeto segue uma série de etapas prédefinida, saindo da Apresentação da Proposta, passando pela Aprovação, Execução, Readequações, Prestação de Contas, e Conclusão.

Penso que ilustrar a etapa, e quem sabe as subetapas, pode dar um ar de dinamismo para a informação. Quem sabe incluímos uma barra de progresso horizontal bem nessa parte:
![captura de tela de 2017-05-21 17-55-07](https://cloud.githubusercontent.com/assets/1244443/26287427/9b5b0662-3e51-11e7-883b-bfc5d115c548.png)
Claro que vcs terão ideias melhores de como ilustrar.. rsrs

Se acharem interessante, posso levantar mais informações sobre essas etapas

**Comment by mateuswetah** [comment]
Sim, acho uma sugestão interessante, podemos ter tipo uma timeline com pontos nestas etapas pré-definidas.

Não há uma ligação direta dessa "etapa" com o status da Situação/Providências? Porque se sim caberia bem ter este dado logo abaixo deles alí.

Isso também volta naquela discussão de conseguirmos relacionar diretamente Proposta e Projeto né....
